client/components/badge/style.scss

```diff
@@ -7,11 +7,11 @@
 }
 
 .badge--warning {
-	color: $gray-dark;
-	background-color: $alert-yellow;
+	color: var( --badge-warning-color );
+	background-color: var( --badge-warning-background-color );
 }
 
 .badge--success {
-	color: $gray-dark;
-	background-color: lighten( $alert-green, 15%);
```

Comment:
not sure about the color change here; might there be a color value which reflects this 15% lighten?

Reply:
I think the Classic Blue (or whatever it's called) should adhere to the new color system if that's the question. We'll have to do some light design work and find an approximate value for colors using sass functions. In this case, it would be great to consolidate and just use --color-success-light instead of this lighten function. Inevitably though, it seems like we'll run into cases where we'll need a color that is not either light or dark but something in between. Not sure what to do at that point.

Reply:
Resolved by switching to `var( --color-success-light )`.